src/rtc/h5_prep.py

```diff
@@ -233,6 +233,26 @@ def save_hdf5_file(hdf5_obj, output_hdf5_file, clip_max,
     logger.info(f'file saved: {output_hdf5_file}')
 
 
+def get_product_version(product_version_runconfig):
+    '''
+    Returns the product version from the product
+    version defined by the user in the runconfig.
+
+    Parameters
+    ---------
+    product_version_runconfig: scalar
+        RunConfig product version
+
+    Returns
+    -------
+    product_version: scalar
+        Product version
+    '''
+    if product_version_runconfig:
+        return product_version_runconfig
+    return SOFTWARE_VERSION
```

Comment:
I think it would be more clear for the user that `SOFTWARE_VERSION` is taken because there if nothing in `product_version_runconfig`. Maybe posting the message to log file?